build/Dockerfile.ci

```diff
@@ -0,0 +1,102 @@
+# CI Dockerfile
+# CI requires a special Dockerfile for use with OpenShift Builds which is what is used to build the image. This requires
+# building the operator from the PR source without using the operator-sdk.
+# NOTE: RUN mkdir <dir> is being used in cases where a directory needs to be created as CI image builds does not seem to
+# be creating the directory as a result of the WORKDIR directive. 
+# build stage for building binaries
+FROM registry.access.redhat.com/ubi8/go-toolset:1.12.8-32 as build
+LABEL stage=build
```

Comment:
I dont understand the labeling here (and elsewhere) what is it used for?

Reply:
It is just for meta-data. As we move to integrating with redhat-operators, the labels here will become more important and will mean something to the ART infrastructure that will build the release version of our container image.

Reply:
Ah okay, could you leave a comment explaining that its used for that? I'm sure others wouldn't know that as well.

Reply:
I will when I add labels needed for ART. This is just a generic label.